RMStore/RMStore.m

```diff
@@ -440,6 +440,15 @@ - (void)paymentQueue:(SKPaymentQueue *)queue updatedDownloads:(NSArray *)downloa
     }
 }
 
+- (BOOL)paymentQueue:(SKPaymentQueue *)queue shouldAddStorePayment:(SKPayment *)payment forProduct:(SKProduct *)product
+{
+    if (!self.appStorePaymentDelegate)
+    {
+        return YES;
```

Comment:
Since there is no delegate and the callback is not handled, should this not return `NO` since the purchase would have to be handled manually later on?

Reply:
If we return NO then the skpayment will be lost and you can't add it again unless the app holds on to it which it can't if it hasn't implemented the delegate. By returning true the payment will be added to the queue by default.